tasks/fork.ts

```diff
@@ -83,18 +83,27 @@ task('fork', 'Forks a chain and starts a JSON-RPC server of that forked chain')
   .setAction(forkNetwork)
 
 subtask('fork:fund-deployer').setAction(async (args, hre) => {
-  const { ethers } = hre
+  const { ethers, network } = hre
 
   const [mainAccount] = await hre.getUnnamedAccounts()
   const { deployer } = await hre.getNamedAccounts()
   if (
     ethers.utils.getAddress(mainAccount) !== ethers.utils.getAddress(deployer)
   ) {
-    await getFunds({
-      to: deployer,
-      tokenSym: 'ETH',
-      amount: hre.ethers.utils.parseEther('1000'),
-      hre,
-    })
+    if (network.name !== 'polygon') {
```

Comment:
I think you meant `===` but also don't forget about **mumbai**

Reply:
fixed